src/header.ce.vue

```diff
@@ -104,40 +108,40 @@ onMounted(() => {
             class="nav-item"
             :class="{ active: props.activeApp === 'datahub' }"
             href="/catalogue/search"
-          >{{ t("catalogue") }}</a
+            >{{ t('catalogue') }}</a
           >
           <a
             class="nav-item"
             :class="{ active: props.activeApp === 'mapstore' }"
             href="/mapstore/"
-          >{{ t("viewer") }}</a
+            >{{ t('viewer') }}</a
           >
           <a
             class="nav-item"
             :class="{ active: props.activeApp === 'mapstore-home' }"
             href="/mapstore/#/home"
-          >{{ t("maps") }}</a
+            >{{ t('maps') }}</a
           >
           <a
             v-if="!isAnonymous"
             class="nav-item"
             :class="{ active: props.activeApp === 'geocontrib' }"
             href="/geocontrib/"
-          >Contributions</a
+            >Contributions</a
```

Comment:
It's not part of your PR, but maybe we could move this label to the translation files as well.